robotiq_hande_driver/hardware/src/hande_hardware_interface.cpp

```diff
@@ -30,26 +44,43 @@ HWI::CallbackReturn RobotiqHandeHardwareInterface::on_init(const HWI::HardwareIn
         rclcpp::get_logger("controller_manager.resource_manager.hardware_"
                            "component.system.RobotiqHandeHardwareInterface"));
 
-    application_layer_.initialize(gripper_position_min_, gripper_position_max_, tty_port_, baudrate_, parity_, data_bits_, stop_bit_, slave_id_);
-
-    RCLCPP_INFO(get_logger(), "Initialized ModbusRTU for %s", tty_port_.c_str());
+    gripper_driver_.initialize(
+        gripper_position_min_,
+        gripper_position_max_,
+        tty_port_,
+        baudrate_,
+        parity_,
+        data_bits_,
+        stop_bit_,
+        slave_id_);
+
+    RCLCPP_INFO(
+        get_logger(),
+        "Initialized ModbusRTU for %s, %d, %c, %d, %d",
+        tty_port_.c_str(),
+        baudrate_,
+        parity_,
+        data_bits_,
+        stop_bit_);
     return HWI::CallbackReturn::SUCCESS;
 }
 
-HWI::CallbackReturn RobotiqHandeHardwareInterface::on_configure(const rlccp_lc::State& /*previous_state*/){
+HWI::CallbackReturn RobotiqHandeHardwareInterface::on_configure(
+    const rlccp_lc::State& /*previous_state*/) {
     int result;
 
-    result = application_layer_.configure();
+    result = gripper_driver_.configure();
```

Comment:
Let's move the `printf(
            "Connecting to: %s, %d, %c, %d, %d\n",
            tty_port_,
            baudrate_,
            parity_,
            data_bits_,
            stop_bit_);` from the `communication.cpp`
```suggestion
    RCLCPP_INFO(get_logger(), "Connecting to tty:%s, baudrate:%d, parity:%c, data_bits:%d, stop_bit:%d", tty_port_.c_str(), baudrate_, parity_, data_bits_, stop_bit_);
    result = gripper_driver_.configure();
```

Reply:
Done.